lib/improver/tests/ensemble_calibration/ensemble_calibration/test_EnsembleCalibration.py

```diff
@@ -1,510 +0,0 @@
-        # generated from using the ensemble mean as the predictor.
-        # In this case, the expectation is for there to be broad agreement
-        # between whether either the ensemble mean or the ensemble
-        # realizations, but the results would not be expected to match exactly.
-        self.assertArrayAlmostEqual(
-            calibrated_predictor.data,
-            self.expected_truncated_gaussian_mean_data, decimal=0)
-        self.assertArrayAlmostEqual(
-            calibrated_variance.data,
-            self.expected_truncated_gaussian_variance_data, decimal=0)
-
-
-@unittest.skipIf(
-    STATSMODELS_FOUND is True, "statsmodels module is available.")
-class Test_process_without_statsmodels(SetupCubes, SetupExpectedResults,
-                                       EnsembleCalibrationAssertions):
-
-    """Additional tests for the process method when the statsmodels module
-    is not available. A simple initial guess will be used for the
-    coefficients that will be solved as part of the calibration."""
-
-    @ManageWarnings(
-        ignored_messages=IGNORED_MESSAGES, warning_types=WARNING_TYPES)
-    def test_gaussian_realizations_data_check(self):
-        """
-        Test that the plugin returns an iris.cube.CubeList
-        of temperature cubes with the expected data, where the plugin
-        returns a cubelist of, firstly, the predictor and, secondly the
-        variance. The ensemble realizations is the predictor.
-        """
-        distribution = "gaussian"
-        predictor_of_mean_flag = "realizations"
-        plugin = Plugin(
-            distribution,
-            predictor_of_mean_flag=predictor_of_mean_flag)
-        calibrated_predictor, calibrated_variance = plugin.process(
-            self.current_temperature_forecast_cube,
-            self.historic_temperature_forecast_cube,
-            self.temperature_truth_cube)
-        self.assertCalibratedVariablesAlmostEqual(
-            calibrated_predictor.data,
-            self.expected_gaussian_realization_no_statsmodels)
-        self.assertCalibratedVariablesAlmostEqual(
-            calibrated_variance.data,
-            self.expected_gaussian_variance_no_statsmodels)
-        # The assertions below are for comparison to the results
-        # generated from using the ensemble mean as the predictor.
-        # In this case, the expectation is for there to be broad agreement
-        # between whether either the ensemble mean or the ensemble
-        # realizations, but the results would not be expected to match exactly.
-        self.assertArrayAlmostEqual(
-            calibrated_predictor.data,
-            self.expected_gaussian_mean_data, decimal=0)
-        self.assertArrayAlmostEqual(
-            calibrated_variance.data,
-            self.expected_gaussian_variance_data, decimal=0)
-
-    @ManageWarnings(
-        ignored_messages=IGNORED_MESSAGES, warning_types=WARNING_TYPES)
-    def test_truncated_gaussian_realizations_data_check(self):
```

Comment:
Same input data as this test:
https://github.com/metoppv/improver/blob/master/lib/improver/tests/ensemble_calibration/ensemble_calibration/test_EstimateCoefficientsForEnsembleCalibration.py#L920